Form/Type/PhoneNumberType.php

```diff
@@ -77,15 +79,38 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'translation_domain' => $options['translation_domain'],
             );
 
+            // To be removed when dependency on Symfony Form is bumped to 2.7.
+            if (!method_exists('Symfony\\Component\\Form\\AbstractType', 'configureOptions')) {
+                $countryChoices = array_flip($countryChoices);
+            } else {
+                $countryOptions['choices_as_values'] = true;
+            }
+
             $countryOptions['required'] = true;
             $countryOptions['choices'] = $countryChoices;
             $countryOptions['preferred_choices'] = $options['preferred_country_choices'];
-            $countryOptions['choice_translation_domain'] = false;
+
+            // Condition to be removed when dependency on Symfony Form is bumped to 2.7.
+            if (method_exists('Symfony\\Component\\Form\\AbstractType', 'configureOptions')) {
+                $countryOptions['choice_translation_domain'] = false;
+            }
+
+            // To be removed when dependency on Symfony Form is bumped to 2.7.
+            if (method_exists('Symfony\\Component\\Form\\FormTypeInterface', 'getName')) {
```

Comment:
This will use the legacy name in 2.8 too. Instead, you can check for the `getBlockPrefix()` method in the `Symfony\Component\Form\AbstractType` class.